### PR TITLE
Add description for getexif() and Exif

### PR DESCRIPTION
**Pull request by greatvovan**
This PR adds description to the above public module members.

### CODE REVIEW

**Comment by radarhere** [comment]
I've created https://github.com/greatvovan/Pillow/pull/1, adding some code examples.

**Comment by greatvovan** [comment]
Thank you for doing all the work @radarhere!

File: src/PIL/Image.py [hunk 1]
```diff
@@ -3601,6 +3606,39 @@ def _apply_env_variables(env=None):
...
+      gps_ifd = exif.get_ifd(ExifTags.IFD.GPSInfo)
+      print(gps_ifd)
+
+    Other IFDs include ``ExifTags.IFD.Exif``, ``ExifTags.IFD.Makernote``,
+    ``ExifTags.IFD.Interop`` and ``ExifTags.IFD.IFD1``.
+
+    :py:mod:`~PIL.ExifTags` also has enum classes to provide names for data::
+
+      print(exif[ExifTags.Base.Software])  # PIL
+      print(gps_ifd[ExifTags.GPS.GPSDateStamp])  # '1999:99:99 99:99:99'
```
Comment (radarhere) [inline src/PIL/Image.py:3639]:
```suggestion
      print(gps_ifd[ExifTags.GPS.GPSDateStamp])  # 1999:99:99 99:99:99
```
Removed quotes that don't appear when printing.

**Review by radarhere — APPROVED** [review]

**Review by hugovk — APPROVED** [review]
Thanks both!

**Pull request merged**